common/client_pool.go

```diff
@@ -65,10 +64,11 @@ func (cp *clientPool) Close() error {
 	for target, cnx := range cp.connections {
 		err := cnx.(*grpc.ClientConn).Close()
 		if err != nil {
-			cp.log.Warn().
-				Str("server_address", target).
-				Err(err).
-				Msg("Failed to close GRPC connection")
+			cp.log.Warn(
+				"Failed to close GRPC connection",
+				slog.String("server_address", target),
+				slog.Any("Error", err),
```

Comment:
nit: should we consider standardizing on `error` as the key for all error messages?

Reply:
yes，i name the key of error as `Error`，and slog didn't provide an convention for it.

Reply:
sorry, i misunderstand it. I will fix it at next time.